packages/mds-trip-processor/src/utils.ts

```diff
@@ -15,24 +15,28 @@ export const eventValidation = (events: TripEvent[], curTime: Timestamp, timeSLA
   return true
 }
 
-export const createTelemetryMap = (events: TripEvent[], tripMap: TripsTelemetry, trip_id: UUID): TripTelemetry[][] => {
+export const createTelemetryMap = (
```

Comment:
Generally speaking, `for` loops and `push` statements are an indication that there is likely a `map`/`reduce` solution.

Reply:
I don't think this case would really benefit from a reduce, we are not using elements of the looped array to push.

Reply:
> I don't think this case would really benefit from a reduce, we are not using elements of the looped array to push.

`forEach`, `filter`, `map`, `reduce` are your friends 😄